src/pages/[marketplace]/deals/index.tsx

```diff
@@ -74,17 +79,17 @@ const Deals: NextPageWithLayout = () => {
 
 		const repaidRatio = new Ratio(principal.toNumber(), principalAmountRepaid.toNumber());
 
-		return repaidRatio.apply(new Big(100));
+		return repaidRatio.apply(100);
```

Comment:
same here, number instead of Big. maybe it's not looking at the right version of the client here?